eng/pipelines/common/global-build-job.yml

```diff
@@ -160,9 +159,6 @@ jobs:
         displayName: Install/build LLVM
 
     - ${{ if eq(parameters.nameSuffix, 'Browser_wasm_Windows') }}:
-=======
-    - ${{ if eq(parameters.platform, 'Browser_wasm_win') }}:
->>>>>>> origin/feature/NativeAOT
```

Comment:
Hmm, is this right? There is no `Browser_wasm_Windows` platform that I can see, only `Browser_wasm_win`.

Should it be the upstream's version, perhaps with an exclusion for not-Mono?

Reply:
I'll take upstream, we needed this certificate update some time back.  It may be ok without it now.  I think at the time, mono took what we did and used the same.